### PR TITLE
Reorg customize config section, add example config and version field …

### PR DESCRIPTION
**Pull request by longmuir**
A few things to close out the Istio operator documentation story. We should make sure to update this as the `Istio` CRD evolves.

### CODE REVIEW

File: bundle/README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ You have access to the OpenShift CLI (oc).
 
 1. When `State: Healthy` appears in the `Status` column, Istio is successfully deployed.
 
+## Selecting the Istio version
+
+The `version` field of the `Istio` custom resource definition defines which version of Istio can be deployed. For a list of available versions, see the [versions.yaml](../versions.yaml) file. This can be set using the `Istio Version` when creating a new `Istio` with the OpenShift Container Platform web console.
```
Comment (luksa) [inline bundle/README.md]: Users can also see the list of versions by running `kubectl explain istio.spec.version`.

File: bundle/README.md [hunk 2]
```diff
@@ -71,6 +96,12 @@ For a list of available configuration for the `values` field, refer to [Istio's
 - [CNI parameters](https://artifacthub.io/packages/helm/istio-official/cni?modal=values)
 - [ZTunnel parameters](https://artifacthub.io/packages/helm/istio-official/ztunnel?modal=values)
 
+## Validating the configuration
+
+The Operator validates the contents of the `spec.values` field. When the Operator parses this field, and encounters an unknown value, it generates an error message. To force the Operator to pass an unknown value, use the `spec.rawValues` field.
```
Comment (luksa) [inline bundle/README.md]:
```suggestion
The Kubernetes API server validates the contents of the `spec.values` field. When it encounters an unknown value, it generates an error message. To pass an unknown value, use the `spec.rawValues` field.
```

File: bundle/README.md [hunk 3]
```diff
@@ -71,6 +96,12 @@ For a list of available configuration for the `values` field, refer to [Istio's
...
+
+You can use the `oc explain` command to display the valid values for the fields in the Istio Custom Resoure Definition (CRD) file. For example, `oc explain istio.spec.values`.
```
Comment (luksa) [inline bundle/README.md]: Should we use `kubectl` instead of `oc` in all of our instructions, since we support vanilla k8s?

Comment (rh-tokeefe) [reply bundle/README.md]: @luksa based on today's team discussion, I created OSSM-5431 to change oc commands to kubectl commands.

File: bundle/README.md (hunk 1 above)
Comment (longmuir) [reply bundle/README.md]: Thanks, I'll add that too.

**Review by luksa — APPROVED** [review]

**Pull request merged**